tutoraspects/plugin.py

```diff
@@ -72,9 +72,14 @@
         ######################
         # ClickHouse Settings
         ("CLICKHOUSE_HOST", "clickhouse"),
-        ("CLICKHOUSE_PORT", "9000"),
+        ("CLICKHOUSE_CLIENT_PORT", "9000"),
```

Comment:
Can we make this `CLICKHOUSE_NATIVE_PORT` and have a secure version of it that [defaults to 9440](https://clickhouse.com/docs/en/guides/sre/network-ports)? Then in the `CLICKHOUSE_SECURE_CONNECTION` it can choose like we do for https.

Reply:
this is better than the old version of having to manually update the clickhouse port option